ui/apps/ui/src/app/components/recommendations/recommendations.component.ts

```diff
@@ -78,4 +83,8 @@ export class RecommendationsComponent implements OnInit {
       filtersConfigs
     );
   }
+
+  _joinDescription(desc: string | string[]): string {
```

Comment:
This function is public - it should not have `_` prefix. Additionally, it would be better to also create pipe so that template does not call it on every redraw cycle.

Reply:
I changed to pipe